src/app/clusters/operational-credentials-server/operational-credentials-server.cpp

```diff
@@ -226,81 +221,25 @@ void emberAfPluginOperationalCredentialsServerInitCallback(void)
     writeFabricsIntoFabricsListAttribute();
 }
 
-// TODO: Use FabricIndex as a parameter instead of fabricId/nodeId/vendorId once AddOptCert + FabricIndex are implemented
 bool emberAfOperationalCredentialsClusterRemoveFabricCallback(EndpointId endpoint, app::CommandHandler * commandObj,
                                                               FabricId fabricId, NodeId nodeId, uint16_t vendorId)
 {
     emberAfPrintln(EMBER_AF_PRINT_DEBUG, "OpCreds: RemoveFabric"); // TODO: Generate emberAfFabricClusterPrintln
 
     EmberAfStatus status = EMBER_ZCL_STATUS_SUCCESS;
-    FabricInfo * fabric;
-    FabricIndex fabricIndex;
-    CHIP_ERROR err = CHIP_NO_ERROR;
-
-    // Fetch matching fabric
-    fabric = GetGlobalFabricTable().FindFabricForNode(fabricId, nodeId, vendorId);
-    VerifyOrExit(fabric != nullptr, status = EMBER_ZCL_STATUS_SUCCESS); // Fabric has already been removed
-
-    // Delete fabric
-    fabricIndex = fabric->GetFabricIndex();
-    err         = GetGlobalFabricTable().Delete(fabricIndex);
-    VerifyOrExit(err == CHIP_NO_ERROR, status = EMBER_ZCL_STATUS_FAILURE);
-
-exit:
+    // TODO - Update RemoveFabric command to take FabricIndex as input parameter
```

Comment:
Let's do this now if possible.

Reply:
This requires the update to cluster definition. The original PR had the change in it, but it was suggested to move it out to a separate PR. It's a trivial change once this PR merges, and I intend to push it right away, since I have the updated cluster definition handy.